vsts/release/package-maven-artifacts-for-release.ps1

```diff
@@ -0,0 +1,199 @@
+# This script builds all of the maven artifacts needed to release a java package
+
+param (
+ [string]$deps,
+ [string]$securityProvider,
+ [string]$tpmProvider,
+ [string]$x509Provider,
+ [string]$provisioningDeviceClient,
+ [string]$provisioningServiceClient,
+ [string]$iotDeviceClient,
+ [string]$iotServiceClient,
+ [string]$defaultWorkingDirectory
+ )
+ 
+function GetVersion($Sources, $artifactId) {
+    # reads a version number from a pom.xml file
+	$pomPath = Join-Path $env:sources "pom.xml"
+	$propertyName = $artifactId
+	$previewTag = "-preview";
+	if ($artifactId.Contains($previewTag))
+	{
+		$propertyName = $artifactId.subString(0, ($artifactId.length - $previewTag.length))
+	}
+	
+	$propertyName = $propertyName + "-version"
+
+	$xml = [xml]$(Get-Content $pomPath -ErrorAction Stop)
+    $version = $xml.project.properties.$propertyName
```

Comment:
You have some tabbing issues here. If you changed it all to spaces, it would render better in the web, too.